RequestManagementSystem/DB/RequestDB.py

```diff
@@ -575,7 +575,10 @@ def getRequestSummaryWeb(self, selectDict, sortList, startItem, maxItems):
 
           if key == 'Type':
             summaryQuery = summaryQuery.join(Request.__operations__)\
-                                       .group_by(Request.RequestID, Operation.Type)
+                                       .group_by(Request.RequestID, Request.RequestName,
+                                                 Request.JobID, Request.OwnerDN, Request.OwnerGroup,
+                                                 Request._Status, Request.Error,
+                                                 Request._CreationTime, Request._LastUpdate)
```

Comment:
The grouping by `Operarion.Type` is gone? why?

Reply:
it should stay clearly

Reply:
Yes. Just fixed.